### PR TITLE
Added different hair to choose for Avatar

### PR DESCRIPTION
**Pull request by AkankshaBodhankar**
Four hair colors have been added for the avatar
hair2
hair3
hair4
hair5

**# Running on app**

screenshot_2016-03-02-21-52-40_powerup systers com powerup
screenshot_2016-03-02-21-52-51_powerup systers com powerup
screenshot_2016-03-02-21-52-55_powerup systers com powerup

### CODE REVIEW

File: PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java [hunk 1]
```diff
@@ -130,6 +134,8 @@ public void onClick(View v) {
 			}
 		});
 
+
+
```
Comment (chhavip) [inline PowerUp/app/src/main/java/powerup/systers/com/AvatarRoomActivity.java]: @AkankshaBodhankar  Remove these extra empty lines

**Comment by AkankshaBodhankar** [comment]
@chhavip Removed the extra empty lines and made a commit

File: PowerUp/.idea/misc.xml [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+      <value>
+        <list size="4">
+          <item index="0" class="java.lang.String" itemvalue="org.jetbrains.annotations.NotNull" />
+          <item index="1" class="java.lang.String" itemvalue="javax.annotation.Nonnull" />
+          <item index="2" class="java.lang.String" itemvalue="edu.umd.cs.findbugs.annotations.NonNull" />
+          <item index="3" class="java.lang.String" itemvalue="android.support.annotation.NonNull" />
+        </list>
+      </value>
+    </option>
+  </component>
```
Comment (chhavip) [inline PowerUp/.idea/misc.xml]: Do not commit your .idea files

File: PowerUp/app/app.iml [hunk 3]
```diff
@@ -65,30 +65,26 @@
...
-      <excludeFolder url="file://$MODULE_DIR$/build/intermediates/bundles" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/classes" />
-      <excludeFolder url="file://$MODULE_DIR$/build/intermediates/coverage-instrumented-classes" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/dependency-cache" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/dex" />
-      <excludeFolder url="file://$MODULE_DIR$/build/intermediates/dex-cache" />
+      <excludeFolder url="file://$MODULE_DIR$/build/intermediates/exploded-aar/com.akexorcist/RoundCornerProgressBar/2.0.3/jars" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/exploded-aar/com.android.support/appcompat-v7/23.0.1/jars" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/exploded-aar/com.android.support/support-v4/23.0.1/jars" />
       <excludeFolder url="file://$MODULE_DIR$/build/intermediates/incremental" />
```
Comment (chhavip) [inline PowerUp/app/app.iml]: Dont commit your iml files

**Comment by chhavip** [comment]
@AkankshaBodhankar I have added some comments, try not to add your iml and idea files as they are user specific, also try to have a single commit in a PR.

**Comment by AkankshaBodhankar** [comment]
@chhavip Will change the commits and make another PR resolving all that you have said :-)

**Comment by aanchal07** [comment]
Unresolved conflicts.

**Pull request closed without merging**